lib/rubocop_todo_corrector/commands/apply.rb

```diff
@@ -57,16 +57,11 @@ def autocorrect_excluded_files
         ::Kernel.system(
           { 'BUNDLE_GEMFILE' => @temporary_gemfile_path },
           [
-            'bundle exec rubocop --force-exclusion', autocorrect_option, *excluded_file_paths
+            'bundle exec rubocop --force-exclusion', autocorrect_option, paths_option
```

Comment:
The main part of this change is here.

Until now, the implementation assumed it would always be present and called `excluded_paths`, raising an exception if it couldn’t be found during the lookup. This has now been changed so that the appropriate branching is handled inside `#paths_options`. If it cannot be found, `nil` is returned, and autocorrection will be executed without arguments.